recipes/example/meta.yaml

```diff
@@ -1,60 +0,0 @@
-# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
```

Comment:
Yep, happens to me sometimes too. We should improve the logic there. Would you mind opening an issue here, @ericdill?